Multisig/UI/UI Library/UIFont+Styles.swift

```diff
@@ -34,7 +34,7 @@ extension GNOTextStyle {
 
     static let footnote2 = GNOTextStyle(size: 13, weight: .medium, color: .gnoDarkGrey)
     static let footnote3 = GNOTextStyle(size: 13, weight: .medium, color: .gnoDarkBlue)
-    static let caption1 = GNOTextStyle(size: 13, weight: .bold)
+    static let caption1 = GNOTextStyle(size: 13, weight: .bold, letterSpacing: 2, color: .gnoMediumGrey)
```

Comment:
this will change all caption1 styles across the app. If you need to change just in your place, please use the builder methods - color() to change color, and if it doesn't exist, please create letterSpacing() method to change it

Reply:
Before I do this I have checked, caption1 is used only in two places, BasicHeaderView which I intend to change and SegmentView which by default use this style also, and when selected it change the color.

Reply:
ok!